test/System/Clock.test.cpp

```diff
@@ -17,21 +19,53 @@ TEST_CASE("[System] sf::Clock")
         STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Clock>);
     }
 
+    SECTION("Construction")
+    {
+        const sf::Clock clock;
+        CHECK(clock.isRunning());
+        CHECK(clock.getElapsedTime() >= sf::microseconds(0));
+    }
+
     SECTION("getElapsedTime()")
     {
         const sf::Clock clock;
         CHECK(clock.getElapsedTime() >= sf::microseconds(0));
         const auto elapsed = clock.getElapsedTime();
-        std::this_thread::sleep_for(std::chrono::milliseconds(1));
+        std::this_thread::sleep_for(1ms);
         CHECK(clock.getElapsedTime() > elapsed);
     }
 
+    SECTION("start()")
+    {
+        sf::Clock clock;
+        clock.start();
+        CHECK(clock.isRunning());
+    }
```

Comment:
There's no test case for the behavior of:

- Stop a running clock
- Take measure of the elapsed time
- Start a stopped clock
- Elapsed time isn't reset